### DISCUSSION TITLE
D9 Connected Fail

### DISCUSSION THREAD

**Discussion by Kap3** [Tasshack/dreame-vacuum#288] [General]
Hi, I'installed HACS plugin. When I add my D9, with user/pass and Region in find my d9 but after I find this error "Failed To Set Up":

![immagine](https://github.com/Tasshack/dreame-vacuum/assets/32192829/1d480341-fe0f-4556-9b15-19fe6063754f)

Can you help me?
Thanks

**Comment by Tasshack** [comment]
Probably because of this https://github.com/Tasshack/dreame-vacuum/issues/256. You can try to install the Beta version.

**Comment by Kap3** [comment]
Thanks! With beta it's ok!
